sycl/test/regression/back_to_back_collectives.cpp

```diff
@@ -0,0 +1,64 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple %s -o %t.out
+// RUN: %HOST_RUN_PLACEHOLDER %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
```

Comment:
This should be located inside [sycl/test/on-device](https://github.com/intel/llvm/tree/sycl/sycl/test/on-device) directory according to the guidelines from [Get Started Guide](https://github.com/intel/llvm/blob/sycl/sycl/doc/GetStartedGuide.md#test-dpc-toolchain).

Reply:
Oops.  I'll go and read the new guidelines.

Reply:
I think the test should be even moved to https://github.com/intel/llvm-test-suite